### PR TITLE
Removed write_human_readable_statepoint.

### PR DESCRIPTION
**Pull request by seoulfood**
## Description

We removed a dead function, deprecated in 0.6.

## Motivation and Context

Related to #395 

## Types of Changes

- [ ] Documentation update
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change<sup>1</sup>

<sup>1</sup>The change breaks (or has the potential to break) existing functionality.

## Checklist:

- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac-flow/blob/master/ContributorAgreement.md).
- [x] My name is on the [list of contributors](https://github.com/glotzerlab/signac-flow/blob/master/contributors.yaml).
- [x] My code follows the [code style guideline](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md#code-style) of this project.
- [x] The changes introduced by this pull request are covered by existing or newly introduced tests.

If necessary:
- [ ] I have updated the API documentation as part of the package doc-strings.
- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
- [ ] I have updated the [changelog](https://github.com/glotzerlab/signac-flow/blob/master/changelog.txt).

### CODE REVIEW

**Review by atravitz — APPROVED** [review]
LGTM!

**Review by mariaward10 — APPROVED** [review]
Looks great!

File: contributors.yaml [hunk 1]
```diff
@@ -129,4 +129,8 @@ contributors:
     given-names: Ramanish
     orcid: "https://orcid.org/0000-0002-7908-060X"
     affiliation: "University of Minnesota"
+  -
+    family-names: Jones
+    given-names: Gabrielle
+    affiliation: "University of Michigan"
```
Comment (mariaward10) [inline contributors.yaml:135]:
```suggestion
    orcid: "https://orcid.org/0000-0002-8041-4873"
    affiliation: "University of Michigan"
```

**Comment by bdice** [comment]
Thanks @seoulfood @mariaward10 @AlainKadar @atravitz @kidrahahjo @DomFijan for joining the session today! I hope you all enjoyed it and learned something new.

I committed the suggestion that @mariaward10 made (adding an ORCID) and I added a line to the changelog for this pull request. This PR can be merged when tests pass.

**Pull request merged**